### PR TITLE
refactor: optimize performance of `queryBtcTimeLockTxByBtcTx()`

### PR DESCRIPTION
**Pull request by ShookLyngs**
## Changes
- Optimize the performance of `RgbppCollector.queryBtcTimeLockTxByBtcTxId()`
    - Refer to PR for more explanations: https://github.com/ckb-cell/rgbpp-explorer/pull/85
    - Source file: https://github.com/ckb-cell/rgbpp-explorer/blob/86fd417eaecc6da626939bab34bf348a6e79cd1e/backend/src/modules/rgbpp/transaction/transaction.service.ts#L214-L246

- resolve https://github.com/ckb-cell/btc-assets-api/issues/45

### CODE REVIEW

**Review by ahonn — APPROVED** [review]
LGTM

File: src/services/rgbpp.ts [hunk 1]
```diff
@@ -270,37 +268,41 @@ export default class RgbppCollector extends BaseQueueWorker<IRgbppCollectRequest
...
-          args: '0x',
-        },
-        scriptType: 'lock',
-        groupByTransaction: true,
-      },
-      {
-        order: 'asc',
-      },
-    );
+  public async queryBtcTimeLockTxByBtcTxId(btcTx: Transaction) {
```
Comment (Flouse) [inline src/services/rgbpp.ts:271]:
```suggestion
  public async queryBtcTimeLockTxByBtcTx(btcTx: Transaction) {
```

**Review by Flouse — CHANGES_REQUESTED** [review]

File: src/services/rgbpp.ts (hunk 1 above)
Comment (ShookLyngs) [reply src/services/rgbpp.ts]: Renamed at 71d3d59

**Comment by Dawn-githup** [comment]
### Performance Bottleneck Optimization Verification (L1—L2)

**Before Optimization:**
- 745 ms
- 755 ms
- 635 ms
- 602 ms
- 580 ms

**After Optimization:**
- 335 ms
- 332 ms
- 349 ms
- 340 ms
- 337 ms

**Review by Dawn-githup — APPROVED** [review]

**Pull request merged**